Server/JokesOnYou.Web.Api/Controllers/AccountController.cs

```diff
@@ -0,0 +1,102 @@
+﻿using JokesOnYou.Web.Api.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Identity;
+using JokesOnYou.Web.Api.Data;
+using JokesOnYou.Web.Api.Services.Interfaces;
+using JokesOnYou.Web.Api.DTOs;
+using JokesOnYou.Web.Api.Repositories.Interfaces;
+
+namespace JokesOnYou.Web.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
+    public class AccountController : ControllerBase
+    {
+        private readonly UserManager<User> _userManager;
+        private readonly SignInManager<User> _signInManager;
+        private readonly ITokenService _ITokenService;
+        private readonly IUserRepository _IUserRepository;
+
+        public AccountController(
```

Comment:
Very subjective opinion but id have theese on prolly ~2 lines since they are not really that important